src/components/toast-notifications/toast-notifications.html

```diff
@@ -0,0 +1,19 @@
+<div class="toast-notifications-list-pf">
+  <div ng-repeat="(alertID, alert) in (alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden"
+       ng-mouseenter="setHover(alert, true)" ng-mouseleave="setHover(alert, false)">
+    <div class="toast-pf alert {{alert.type | alertStatus}}" ng-class="{'alert-dismissable': !hideCloseButton}">
+      <button ng-if="!hideCloseButton" type="button" class="close" ng-click="close(alert)" aria-hidden="true">
+        <span class="pficon pficon-close"></span>
+      </button>
+      <span class="{{alert.type | alertIcon}}" aria-hidden="true"></span>
+      <span class="sr-only">{{alert.type}}</span>
+      <span class="'toast-alert-message" ng-if="alert.message" style="margin-right: 5px;">{{alert.message}}</span>
```

Comment:
we have classes for common margin sizes, these may not have been moved to common yet, but these are things like mar-right-sm (which is 5px right margin)